imageprocessor/processorcommand/gm.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/Imgur/mandible/imageprocessor/thumbType"
 )
 
-const GM_COMMAND = "convert"
+const GM_COMMAND = "gm convert"
```

Comment:
I don't think the exec command will like this space. You might need to change the `runProcessorCommand` so that it adds `convert` as the first item in the argument slice